network/p2p/unicast/ratelimit/internal/limiter_map/rate_limiter_map.go

```diff
@@ -0,0 +1,109 @@
+package limiter_map
+
+import (
+	"sync"
+	"time"
+
+	"golang.org/x/time/rate"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+type RateLimiterMetadata struct {
+	// Limiter the rate Limiter
+	Limiter *rate.Limiter
+	// LastRateLimit the last timestamp this the peer was rate limited.
+	LastRateLimit time.Time
+	// LastAccessed the last timestamp when the Limiter was used. This is used to Cleanup old Limiter data
+	LastAccessed time.Time
```

Comment:
These are only used in this module and don't need to be exported
```suggestion
	// lastAccessed the last timestamp when the Limiter was used. This is used to Cleanup old Limiter data
	lastAccessed time.Time
```